src/StatsigServer.ts

```diff
@@ -554,7 +554,7 @@ function trimUserObjIfNeeded(user: StatsigUser | null): StatsigUser {
     user.userID = user.userID.toString().substring(0, MAX_VALUE_SIZE);
   }
   if (shouldTrimParam(user, MAX_USER_SIZE)) {
-    user.custom = {};
+    user.custom = { statsig_error: 'User object length too large' };
```

Comment:
The other interesting case here is that this is too large because of the customIDs field.  So clearing custom might not necessarily solve this

Reply:
@tore-statsig do you think we should add another step where we first clear custom, then clear customIDs, then clear the whole user object?